### PR TITLE
Right language

### PR DESCRIPTION
**Pull request by TonySchroeder**
close https://github.com/progwise/george-ai/issues/39

### CODE REVIEW

File: apps/playwright-scraper/src/chat-gpt.ts [hunk 1]
```diff
@@ -8,19 +8,27 @@ const configuration = new Configuration({
 })
 const openai = new OpenAIApi(configuration)
 
-export const getServiceSummary = async (content: string) => {
+export const getServiceSummary = async (content: string, language: string) => {
   try {
+    let instruction = ''
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/chat-gpt.ts:13]: and if language is not in de, en ?

File: apps/playwright-scraper/src/chat-gpt.ts [hunk 2]
```diff
@@ -8,19 +8,21 @@ const configuration = new Configuration({
...
   try {
     const response = await openai.createChatCompletion({
       model: 'gpt-3.5-turbo',
       messages: [
         {
           role: 'system',
           content:
-            'Gib mir eine Zusammenfassung der Angebote in maximal 300 Worten für folgenden Textinhalt einer Website mit Nennung von Ansprechpartnern und Kontaktinformationenen.',
+            language === 'de'
+              ? 'Gib mir eine Zusammenfassung der Angebote in maximal 300 Worten für folgenden Textinhalt einer Website mit Nennung von Ansprechpartnern und Kontaktinformationenen, in Deutsch.'
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/chat-gpt.ts:20]: something like prompts? 
```
const prompts: {
  de: {
    summary: [
      'Gib mir eine Zusammenfassung der Angebote in maximal 300 Worten für folgenden Textinhalt einer Website mit Nennung von Ansprechpartnern und Kontaktinformationenen, in Deutsch.',
      'Formatiere die Antwort'
    ],
    keywords: []
  },
  en: {
    summary: [
      'Provide a summary of the offerings in a maximum of 300 words for the following website content, mentioning the contact persons and contact information, in english.',
      'Format the answer'

    ],
    keywords: []
  }
}
```

Comment (TonySchroeder) [reply apps/playwright-scraper/src/chat-gpt.ts]: The texts for AI requests have been consolidated in a prompts variable, and Language types have been converted into an enum.

File: apps/playwright-scraper/src/chat-gpt.ts [hunk 3]
```diff
@@ -8,63 +13,56 @@ const configuration = new Configuration({
 })
 const openai = new OpenAIApi(configuration)
 
-export const getServiceSummary = async (content: string) => {
+export enum Language {
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/chat-gpt.ts:16]: It is too complex imho.
I do not like enums if I do not need them and here I do not need.

File: apps/playwright-scraper/src/chat-gpt.ts [hunk 4]
```diff
@@ -8,63 +13,56 @@ const configuration = new Configuration({
...
+  DE = 'de',
+  EN = 'en',
+}
+
+export const getServiceSummary = async (
+  content: string,
+  language: Language,
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/chat-gpt.ts:23]:
```suggestion
  language: 'de' | 'en',
```

File: apps/playwright-scraper/src/chat-gpt.ts [hunk 5]
```diff
@@ -8,63 +13,56 @@ const configuration = new Configuration({
...
-          "Die List soll folgendes Format haben: 'Keyword1, Keyword2, Keyword3, ..., Keyword10'.",
-      },
-      {
-        role: 'system',
-        content: 'Antworte nur mit den Keywords, ohne ein Prefix.',
-      },
-      { role: 'user', content },
-    ],
-  })
+export const getKeywords = async (content: string, language: Language) => {
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/chat-gpt.ts:45]:
```suggestion
export const getKeywords = async (content: string, language: 'de' | 'en') => {
```

File: apps/playwright-scraper/src/scrape.ts [hunk 6]
```diff
@@ -1,11 +1,12 @@
 import playwright, { Page } from 'playwright-chromium'
+import { Language } from './chat-gpt'
 
 export interface ScrapeResult {
   title: string
   url: string
   content: string
   links: string[]
-  language: string
+  language: Language
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/scrape.ts:9]: string - you want to now the real found language and not what we can prompt for. The fallback should be implemented in the chatGpt implementation.

File: apps/playwright-scraper/src/scrape.ts [hunk 7]
```diff
@@ -5,7 +5,7 @@ export interface ScrapeResult {
   url: string
   content: string
   links: string[]
-  language: string
+  language: 'de' | 'en'
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/scrape.ts:8]:
```suggestion
  language: string
```

File: apps/playwright-scraper/src/scrape.ts [hunk 8]
```diff
@@ -55,9 +55,13 @@ export const scrapePage = async (
     '\n\n' +
     // eslint-disable-next-line unicorn/prefer-string-replace-all
     allTexts.map((text) => text.replace(/\s\s+/g, ' ')).join('\n')
-
-  const language = (await page.locator('html').getAttribute('lang')) || ''
+  const detectedLanguage =
+    (await page.locator('html').getAttribute('lang')) || 'en'
+  const language = ['de', 'en'].includes(detectedLanguage)
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/scrape.ts:60]: no fallback in scrape

File: apps/playwright-scraper/src/scrape.ts [hunk 9]
```diff
@@ -55,9 +55,13 @@ export const scrapePage = async (
     '\n\n' +
     // eslint-disable-next-line unicorn/prefer-string-replace-all
     allTexts.map((text) => text.replace(/\s\s+/g, ' ')).join('\n')
-
-  const language = (await page.locator('html').getAttribute('lang')) || ''
+  const detectedLanguage =
```
Comment (moncapitaine) [inline apps/playwright-scraper/src/scrape.ts:58]: I would trim

**Pull request merged**